com.heroicgameslauncher.hgl.yml

```diff
@@ -122,6 +122,9 @@ modules:
         commit: 4dc99dff76218718763a6b07fc1900fa6d1dafd9
         x-checker-data:
           type: git
+      - type: patch
+        paths:
+          - patches/gamemode/disable-pidfd.patch
```

Comment:
patch is required to compile because newer compilers have removed pidfd functions, it's a linux syscall so it's easily added back in the gamemode code through a basic syscall wrapper if the function doesn't exist